net/dnscrypt-proxy/files/dnscrypt-resolvers.csv

```diff
@@ -1,7 +1,10 @@
 Name,Full name,Description,Location,Coordinates,URL,Version,DNSSEC validation,No logs,Namecoin,Resolver address,Provider name,Provider public key,Provider public key TXT record

```

Comment:
This separated package (dnscrypt-proxy-resolvers) has only one file — this (dnscrypt-resolvers.csv).